traversal/procedure/GraphProcedure.java

```diff
@@ -194,8 +205,6 @@ private void registerVertex(PlannerVertex<?> plannerVertex, Set<PlannerVertex<?>
             List<PlannerVertex<?>> adjacents = new ArrayList<>();
 
             ProcedureVertex<?, ?> vertex = vertex(plannerVertex);
-            if (vertex.isThing()) vertex.asThing().props(plannerVertex.asThing().props());
-            else vertex.asType().props(plannerVertex.asType().props());
```

Comment:
unrelated: move settings `props` into the vertex builder methods below -- this is more canonical and reduces one `if` here that was already being performed in the vertex builder